server/app/views/dev/DatabaseSeedView.java

```diff
@@ -99,6 +99,7 @@ public Content render(Request request, Optional<String> maybeFlash) {
                     .withClasses("flex", "flex-col", "gap-4")
                     .with(createSeedSection(request))
                     .with(createCachingSection(request))
+                    .with(createIconsSection())
```

Comment:
Do we also want to add a comment in `Icons.java` pointing people to the icons page?

Reply:
Maybe later, but I'm not going to bother right now.